sequencer/src/persistence/sql.rs

```diff
@@ -363,15 +363,16 @@ impl SequencerPersistence for Persistence {
         let json = serde_json::to_value(cfg)?;
 
         let mut tx = self.db.write().await?;
-        tx.execute_one_with_retries("INSERT INTO network_config (config) VALUES ($1)", (json,))
```

Comment:
@imabdulbasit not sure what to do here. Would just lifting the old code for `execute_one_with_retries` here be sufficient?